tsconfig.json

```diff
@@ -1,6 +1,12 @@
 {
   "compilerOptions": {
     "module": "commonjs",
+    "lib": [
+      "DOM",
+      "ES5",
+      "ScriptHost",
```

Comment:
What references is irrelevant here - we shouldn't be including any libraries we don't have. We shouldn't even include `dom` since we're not in the DOM. All that does is give a false sense of security and errors at runtime instead.